WooCommerce/Classes/ViewRelated/Products/Edit Product/Product Settings/ProductSettingsRows.swift

```diff
@@ -102,10 +102,12 @@ enum ProductSettingsRows {
         }
 
         func handleTap(sourceViewController: UIViewController, onCompletion: @escaping (ProductSettings) -> Void) {
+            /// Check if the store is eligible for password editing or is authenticated with WPCom.
+            /// If neither condition is met, the cell will not be selectable.
```

Comment:
I tested on Android and found that they allow visibility editing with 2 options public and private. Should we follow the same? If not, maybe it's better to hide the row entirely or display "Not editable" instead of "Public". The current experience makes it look like a bug, with the row tappable but no navigation.

Reply:
`Public` here is the "visibility" of the product, so I think it's better to display `Public` instead of `Not Editable`.
So, probably here the solution is something in the middle: display a small alert that explain to the user the status and why they cannot edit it. Wdyt?

Reply:
Yup, that sounds like a better solution. Please feel free to take care of it in a separate PR.